.github/workflows/ci-wheels.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     - name: "build ${{ matrix.os }} (${{ matrix.arch }}) wheels"
       uses: pypa/cibuildwheel@v2.20.0
       env:
-        CIBW_SKIP: "cp39-* cp313-* pp* *-musllinux*"
+        CIBW_SKIP: "cp313-* pp* *-musllinux*"
```

Comment:
So we were previously skipping Python 3.9, but now we're not? That is my interpretation of [the docs](https://cibuildwheel.pypa.io/en/stable/options/#build-skip). This doesn't add up to me. Can you explain this to me?